src/routes/api/client-key/recover-client-key.ts

```diff
@@ -94,7 +94,7 @@ const getAndVerifyplainClientKey = async ({ id, jiraHost, clientKey: hashedClien
 		//https://hello.atlassian.net/wiki/spaces/EDGE/pages/315794033/Whitelist+Proxy+-+Usage
 		//use whitelist proxy so that we can hit xxx.jira-dev.com
 		//Unless we don't care about those records in db?
-		const proxy = `http://${process.env.WHITELIST_PROXY_HOST}:${process.env.WHITELIST_PROXY_PORT}`;
+		const proxy = `http://${process.env.WHITELIST_PROXY_HOST ?? ""}:${process.env.WHITELIST_PROXY_PORT ?? ""}`;
```

Comment:
supernit: why not "undefined"? 🤣  doesn't matter as I think it doesn't matter, but in other places it is "undefined"